IdentityCore/src/requests/broker/mac/MSIDSSOXpcInteractiveTokenRequest.m

```diff
@@ -96,13 +96,26 @@ - (void)performXpcRequest:(NSDictionary *)xpcRequest
 
     // Retrieve the bundle identifier
     NSString *bundleIdentifier = [mainBundle bundleIdentifier];
+
+    // Use parentViewController frame if available, otherwise fall back to presentationAnchorWindow frame.
+    // This is also aligned when providing anchor window for the SSO Extension
+    CGRect xpcAnchorFrame = CGRectZero;
+    MSIDViewController *strongParentViewController = self.requestParameters.parentViewController;
```

Comment:
just curious why name is strong?

Reply:
The `parentViewController` is declared in the original class.

`@property (nonatomic, weak) MSIDViewController *parentViewController;`

And I will need to create a local strong to it when reference other system will through error as weak reference may be deallocating why referencing